sql/parser2/scan_test.go

```diff
@@ -40,11 +40,9 @@ func TestScanner(t *testing.T) {
 		{`>`, []int{'>'}},
 		{`>=`, []int{GREATER_EQUALS}},
 		{`=`, []int{'='}},
-		{`=>`, []int{EQUALS_GREATER}},
```

Comment:
`=>` and `:=` were both allowed by the postgres grammar for the specification of named arguments to custom functions. This wasn't mentioned in the docs, but can be seen from the original grammar: https://github.com/postgres/postgres/blob/master/src/backend/parser/gram.y#L12936